### PR TITLE
Check video duration before generate thumbnails

### PR DESCRIPTION
**Pull request by agombault**
This change add a check on video duration before generating thumbnails.
Prevent generating an empty thumbnails sprite.

https://github.com/G-Corp/vice/issues/12

### CODE REVIEW

**Comment by Kamasoutra** [comment]
🚴

**Comment by glejeune** [comment]
@agombault I'll add a test before merging ;)

File: src/vice_thumbnails.erl [hunk 1]
```diff
@@ -70,31 +70,40 @@ generate(Movie, OutName, Options, Fun) ->
...
-              {async, Ref}
+            Error ->
+              Error
           end;
         Error ->
           Error
       end;
+    {ok, Duration} ->
+      error_logger:error_msg("Every parameter is bellow video duration => ~p", [Duration]),
+      error;
```
Comment (glejeune) [inline src/vice_thumbnails.erl]: In case or error, we need to return an error tuple.
```suggestion
      {error, duration};
```

Comment (agombault) [reply src/vice_thumbnails.erl]: Added ! https://github.com/G-Corp/vice/pull/13/files#diff-fe7b8882d0a6bc67ae95015c86096a41R104

**Comment by agombault** [comment]
Hi @glejeune, I added a small test.
Thanks

**Pull request merged**